src/components/transport_manager/src/transport_adapter/transport_adapter_impl.cc

```diff
@@ -692,6 +692,34 @@ void TransportAdapterImpl::ConnectFailed(const DeviceUID& device_handle,
   LOG4CXX_TRACE(logger_, "exit");
 }
 
+void TransportAdapterImpl::RemoveFinalizedConnection(
+    const DeviceUID& device_handle, const ApplicationHandle& app_handle) {
+  const DeviceUID device_uid = device_handle;
+  const ApplicationHandle app_uid = app_handle;
+  LOG4CXX_TRACE(
+      logger_,
+      "enter. device_id: " << &device_uid << ", app_handle: " << &app_uid);
```

Comment:
See https://adc.luxoft.com/confluence/display/APPLINK/Logger+levels+and+property+files+usage